andela/ah-sealteam-frontend
# #161278871 Users should be able to create articles

#### What does this PR do?
- Adds functionality for users to create and publish articles
#### Description of Task to be completed?
- Authenticated users can create and publish articles
#### How should this be manually tested?
   `$ git clone https://github.com/andela/ah-sealteam-frontend.git`
   `$ git checkout feature/Users-should-be-able-to-view-and-edit-their-profiles-#161278870`
   `$ npm install`
   `$ npm start`
   $ Register account via `http://localhost:3000/signup`
- To create an article navigate to `http://localhost:3000/articles/create`
#### What are the relevant pivotal tracker stories?
[#161278871](https://www.pivotaltracker.com/n/projects/2205273/stories/161278870)
- Worked on by @botnetdobbs & @gtsofa
